magda-content-api/src/schemas.ts

```diff
@@ -50,6 +50,80 @@ export const headerNavigation: any = {
     required: ["order"]
 };
 
+export const footerCategoryItem: any = {
+    type: "object",
+    properties: {
+        order: {
+            type: "number",
+            default: 1
+        },
+        label: {
+            type: "string",
+            minLength: 1
+        },
+        links: {
+            type: "array",
+            items: {
+                type: "array",
+                minItems: 2,
```

Comment:
Why this length limit range?

Reply:
Current config data is a 2 dimension array:
```
"links": [
    [
      "About",
      "page/about"
    ],
    [
      "Suggest a dataset",
      "suggest"
    ]
]
```
The first item is the label and the second item is the link.
`maxItems` used to be 3 --- it was a mistake and I've corrected it